CMP1903_A1_2324/Die.cs

```diff
@@ -28,10 +28,10 @@ public int Roll()
             // creates the random
```

Comment:
does not follow the spacing conventions of the stylistic guidelines. Is a good use of single purpose method.